Package.swift

```diff
@@ -182,6 +182,7 @@ let package = Package(
             name: "Basics",
             dependencies: [
                 "SPMSQLite3",
+                .product(name: "DequeModule", package: "swift-collections"),
```

Comment:
Required for `actor TokenBucket`, which is used below for limiting the amount of concurrent HTTP requests.